src/neptune_scale/types.py

```diff
@@ -63,8 +63,45 @@ def _as_list(arr: ArrayLike[T]) -> list[T]:
 
 @dataclass
 class Histogram:
-    """Represents a histogram with explicit bin edges and counts/densities
-    falling into specific bins.
+    """Represents a histogram with explicit bin edges.
+
+    To specify the data distribution across bins, you can use either counts or densities. Note that n bins have n+1
+    bin edges, so the length of the `bin_edges` argument should be one more than the length of the counts or densities.
+
+    Use the histogram as a value in the dictionary passed to `Run.log_histograms()`.
+
+    Args:
+        bin_edges (ArrayLike): The bin edges of the histogram. The maximum number of edges is 513.
+            To create unbounded bins at the beginning or end, use `inf` values.
+        counts (ArrayLike, optional): Number of data points that fall into each bin.
+        densities (ArrayLike, optional): Density values for each bin.
+            If passing custom values, they must be normalized so that the area of the histogram bars equals 1.
```

Comment:
Just for the record, we don't validate this and I doubt backend does. So maybe we shouldn't have this comment here.